src/core/components/courses/OpenQuestionChart.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <q-card class="card" flat>
+    <div class="text-weight-bold">{{ card.question }}</div>
+    <div class="q-mb-lg subtitle">{{ subtitle }}</div>
+    <div v-for="(answer, index) in card.answers" :key="index" class="q-mt-sm answer-container">
+      <div class="q-ma-sm">{{ answer }}</div>
+    </div>
+  </q-card>
+</template>
+
+<script>
+export default {
+  name: 'OpenQuestionChart',
+  props: {
+    card: { type: Object, default: () => ({}) },
+  },
+  computed: {
+    subtitle () {
+      return `${this.card.answers.length} réponses à cette question ouverte`;
```

Comment:
return `${this.card.answers.length} réponse{{ this.card.answers.length > 1 ? 's' : '' }} à cette question ouverte`;

Reply:
@sophiemoustard Je crois me souvenir que Corentin avait créé une fonction pour gérer le pluriel, ça a été fait ?

Reply:
si tu te pose la question sur ce style de fonction, va voir dans les fichiers `utils.js`. c'est leur place
en l'occurence oui il y a `formatQuantity` que tu peux utiliser Félix

Reply:
daccord merci, elle ny est pas dans `utils` donc je l'ai mise en dur (+ rajoutée sur surveychart)

Reply:
Vu ensemble :
`return `${formatQuantity(réponse, this.card.answers.length)}  à cette question ouverte`;`